opentelemetry-api/src/opentelemetry/metrics/__init__.py

```diff
@@ -178,60 +160,87 @@ def get_handle(self, label_values: List[str]) -> "object":
+    def remove_handle(self, label_values: Tuple[str, ...]) -> None:
         """Removes the handle from the Metric, if present.
 
         The handle with matching label values will be removed.
 
         args:
-            label_values: The list of label values to match against.
+            label_values: The label values to match against.
         """
 
     def clear(self) -> None:
         """Removes all handles from the `Metric`."""
 
 
-class FloatCounter(Metric):
-    """A counter type metric that holds float values."""
-
-    def get_handle(self, label_values: List[str]) -> "CounterHandle":
-        """Gets a `CounterHandle` with a float value."""
+class DefaultMetric(Metric):
+    """The default Metric used when no Metric implementation is available."""
 
+    def get_handle(
+        self, label_values: Tuple[str, ...]
+    ) -> "DefaultMetricHandle":
+        return DefaultMetricHandle()
 
-class IntCounter(Metric):
-    """A counter type metric that holds int values."""
 
-    def get_handle(self, label_values: List[str]) -> "CounterHandle":
-        """Gets a `CounterHandle` with an int value."""
+class Counter(Metric):
+    """A counter type metric that expresses the computation of a sum."""
 
+    def get_handle(self, label_values: Tuple[str, ...]) -> "CounterHandle":
+        """Gets a `CounterHandle`."""
+        return CounterHandle()
 
-class FloatGauge(Metric):
-    """A gauge type metric that holds float values."""
 
-    def get_handle(self, label_values: List[str]) -> "GaugeHandle":
-        """Gets a `GaugeHandle` with a float value."""
+class Gauge(Metric):
+    """A gauge type metric that expresses a pre-calculated value.
 
+    Gauge metrics have a value that is either ``Set`` by explicit
+    instrumentation or observed through a callback. This kind of metric
+    should be used when the metric cannot be expressed as a sum or because
+    the measurement interval is arbitrary.
+    """
 
-class IntGauge(Metric):
-    """A gauge type metric that holds int values."""
+    def get_handle(self, label_values: Tuple[str, ...]) -> "GaugeHandle":
+        """Gets a `GaugeHandle`."""
+        return GaugeHandle()
 
-    def get_handle(self, label_values: List[str]) -> "GaugeHandle":
-        """Gets a `GaugeHandle` with an int value."""
 
+class Measure(Metric):
+    """A measure type metric that represent raw stats that are recorded.
 
-class FloatMeasure(Metric):
-    """A measure type metric that holds float values."""
+    Measure metrics represent raw statistics that are recorded. By
+    default, measure metrics can accept both positive and negatives.
+    Negative inputs will be discarded when monotonic is True.
+    """
 
-    def get_handle(self, label_values: List[str]) -> "MeasureHandle":
+    def get_handle(self, label_values: Tuple[str, ...]) -> "MeasureHandle":
         """Gets a `MeasureHandle` with a float value."""
+        return MeasureHandle()
+
+
+class MetricHandle(ABC):
```

Comment:
An empty interface 😢  Maybe a common base class for the handles does not make sense on the API level (it might help for the implementation on the SDK level of course).